File: src/pcb.cpp
#include "pcb.h"


Pcb::Pcb(){
  this->id = 0;
  this->estimatedTime = 0;
  return 0;
}

Pcb::~Pcb(){
  return 0;
}

int Pcb::getId(){
  return this->id;
}

int Pcb::getEstimatedTime(){
  return this->estimatedTime;
}

void Pcb::setEstimatedTime(int other){
  this->estimatedTime = other;
}

void Pcb::setId(int other){
  this->id = other;
}

File: src/src/pcb/Pcb.java
package pcb;

public class Pcb{

  private Integer Id;
  private Integer EstimatedTime;
  private Integer quantum;

	public Pcb(){
		this.Id = 0;
    	this.EstimatedTime = 0;
    	this.quantum = 0;
  	}


  	public void setId(Integer Id){
    	this.Id = Id;
  	}  

  	public void setEstimatedTime(Integer EstimatedTime){
    	this.EstimatedTime = EstimatedTime;
  	}

  	public void setQuantum(Integer quantum){
    	this.quantum = quantum;
  	}

  	public Integer getId(){
    	return this.Id;
  	}

  	public Integer getEstimatedTime(){
	    return this.EstimatedTime;
	}

	public Integer getQuantum(){
		return this.quantum;
	}


}